package.json

```diff
@@ -175,7 +177,7 @@
     "html-webpack-harddisk-plugin": "2.0.0",
     "html-webpack-plugin": "5.3.2",
     "husky": "4.2.1",
-    "iconscout-unicons-tarball": "https://github.com/grafana/icons/tarball/9728be621a4e7d891611149c9cd179e793f790a7",
+    "iconscout-unicons-tarball": "https://github.com/grafana/icons/tarball/63056cd833ba7ee4e94904492b3a8c0cabc38d28",
```

Comment:
Is this change required for the yarn change?

Reply:
Yes. The previous commit doesn't include a package.json manifest in the project root, which is required with yarn 2.